src/Sentry/TransactionTracer.cs

```diff
@@ -0,0 +1,271 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Sentry
+{
+    /// <summary>
+    /// Transaction tracer.
+    /// </summary>
+    public class TransactionTracer : ITransactionTracer
+    {
+        private readonly IHub _hub;
+
+        // This and many other properties in this object are mere
+        // wrappers around 'contexts.trace'.
+        // It poses danger as the raw contexts object is also exposed
+        // to the user.
+        //
+        // As an example, the user can do something seemingly innocuous like this:
+        //
+        // var t = new Transaction(hub, "my transaction", "my op")
+        // {
+        //     Contexts = new Contexts
+        //     {
+        //         ["my ctx"] = myObj
+        //     }
+        // }
+        //
+        // The above code overwrites the contexts and, as a result, the "my op" value
+        // is lost completely.
```

Comment:
I'm not entirely sure if it's actually helpful to have this in the docs, we already have an issue for that, right?
giving examples of what to NOT do is..a red flag.

Reply:
yeah, this was added before the issue existed.
i will remove it.